spanner/spanner_snippets/snippet.go

```diff
@@ -1266,6 +1277,329 @@ func updateUsingBatchDML(ctx context.Context, w io.Writer, client *spanner.Clien
 
 // [END spanner_dml_batch_update]
 
+// [START spanner_create_table_with_datatypes]
+
+func createTableWithDatatypes(ctx context.Context, w io.Writer, adminClient *database.DatabaseAdminClient, database string) error {
+	op, err := adminClient.UpdateDatabaseDdl(ctx, &adminpb.UpdateDatabaseDdlRequest{
+		Database: database,
+		Statements: []string{
+			`CREATE TABLE Venues (
+				VenueId	INT64 NOT NULL,
+				VenueName STRING(100),
+				VenueInfo BYTES(MAX),
+				Capacity INT64,
+				AvailableDates ARRAY<DATE>,
+				LastContactDate DATE,
+				OutdoorVenue BOOL,
+				PopularityScore FLOAT64,
+				LastUpdateTime TIMESTAMP NOT NULL OPTIONS (allow_commit_timestamp=true)
+			) PRIMARY KEY (VenueId)`,
+		},
+	})
+	if err != nil {
+		return err
+	}
+	if err := op.Wait(ctx); err != nil {
+		return err
+	}
+	fmt.Fprintf(w, "Created Venues table in database [%s]\n", database)
+	return nil
+}
+
+// [END spanner_create_table_with_datatypes]
+
+// [START spanner_insert_datatypes_data]
+
+func writeDatatypesData(ctx context.Context, w io.Writer, client *spanner.Client) error {
+	venueColumns := []string{"VenueId", "VenueName", "VenueInfo", "Capacity", "AvailableDates",
+		"LastContactDate", "OutdoorVenue", "PopularityScore", "LastUpdateTime"}
+	exampleBytes1 := []byte("Hello World 1")
```

Comment:
Do these all need to be separate variables? Or can they all be inlined?

Reply:
Updated.